nexus/lib/qmcpack_input.py

```diff
@@ -1834,17 +1834,23 @@ class sposet(QIxml):
                   'source','version','precision','tilematrix',
                   'meshfactor']
     elements   = ['occupation','coefficient','coefficients']
-    text       = 'spos'
+    #text       = 'spos'
```

Comment:
Why did this need to be commented?

Reply:
This relates to the body text of a qmcpack input element.  Sposets have no body text by default (i.e. not <sposet> spos </sposet>.  It's an odd holdover that doesn't have any effect I can tell, but is formally incorrect.